src/ferui-components/tree-view/tree-view-component.ts

```diff
@@ -81,6 +82,8 @@ export class FuiTreeViewComponent<T> implements OnInit, OnDestroy {
 
   @Input() error: boolean = false;
 
+  @Input() autoSelector?: TreeViewAutoSelector<T>;
```

Comment:
Please rename it into `autoNodeSelector`.

Reply:
Done